### ISSUE TITLE
Building flexprop 6.6.1 on latest RPi OS version: Bookworm, fails in building loadp2.c

### ISSUE THREAD

**Issue by dgately** [totalspectrum/flexprop#85]
Building loadp2.c (either on its own or via flexprop's Makefile) fails in not finding these 2 fpga .bin files. 

MainLoader_fpga_bin
MainLoader_fpga_bin_len

To get the build to complete, knowing that I no longer require fpga loading, I just changed the names of these binary files to: MainLoader_chip_bin & MainLoader_chip_bin_len to complete the compile of loadp2

This issue is probably similar to another issue where P2ES_flashloader.bin & P2ES_sdcard.bin files are not created when building loadp2...

During the build process, I noticed that flexspin had been successfully built and was available in flexprop's bin directory. I made sure that my PATH var included a path to that directory.

Note:
A warning had appeared in earlier attempts to build flexprop, that required installing openspin. Is openspin a requirement to build flexprop? 

```
/loadp2 $ make
flexspin -2 -o MainLoader_chip.bin MainLoader_chip.spin2
Propeller Spin/PASM Compiler 'FlexSpin' (c) 2011-2023 Total Spectrum Software Inc. and contributors
Version 6.6.1-HEAD-v6.5.4-22-g75f48863 Compiled on: Nov  6 2023
MainLoader_chip.spin2
MainLoader_chip.spin2
Done.
Program size is 1024 bytes
xxd -i MainLoader_chip.bin > MainLoader_chip.h
flexspin -2 -o flash_loader.bin flash_loader.spin2
Propeller Spin/PASM Compiler 'FlexSpin' (c) 2011-2023 Total Spectrum Software Inc. and contributors
Version 6.6.1-HEAD-v6.5.4-22-g75f48863 Compiled on: Nov  6 2023
flash_loader.spin2
flash_loader.spin2
Done.
Program size is 496 bytes
xxd -i flash_loader.bin > flash_loader.h
gcc -Wall -O -g  -o build/loadp2 loadp2.c loadelf.c osint_linux.c u9fs/u9fs.c u9fs/authnone.c u9fs/print.c u9fs/doprint.c u9fs/rune.c u9fs/fcallconv.c u9fs/dirmodeconv.c u9fs/convM2D.c u9fs/convS2M.c u9fs/convD2M.c u9fs/convM2S.c u9fs/readn.c
loadp2.c: In function ‘loadfileFPGA’:
loadp2.c:511:25: error: ‘MainLoader_fpga_bin’ undeclared (first use in this function); did you mean ‘MainLoader_chip_bin’?
  511 |     txstring((uint8_t *)MainLoader_fpga_bin, MainLoader_fpga_bin_len);
      |                         ^~~~~~~~~~~~~~~~~~~
      |                         MainLoader_chip_bin
loadp2.c:511:25: note: each undeclared identifier is reported only once for each function it appears in
loadp2.c:511:46: error: ‘MainLoader_fpga_bin_len’ undeclared (first use in this function); did you mean ‘MainLoader_chip_bin_len’?
  511 |     txstring((uint8_t *)MainLoader_fpga_bin, MainLoader_fpga_bin_len);
      |                                              ^~~~~~~~~~~~~~~~~~~~~~~
      |                                              MainLoader_chip_bin_len
make: *** [Makefile:65: build/loadp2] Error 1
rm flash_loader.bin MainLoader_chip.bin

```

**Comment by totalspectrum** [comment]
I wasn't able to reproduce this. Based on the error message, I suspect it is due to an empty MainLoader_fpga.h file created in the build process when xxd was not found. When I started from scratch with xxd installed, flexprop built and ran correctly on a Raspberry Pi 4b.

At one time openspin was required to build the P1 tools, but that should no longer be the case, and I didn't see any warnings during my build (maybe I missed them?). There's only reference I see to openspin is a commented out line in Proploader/Makefile.

Thank you for your bug reports!
